src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let number = 0;
+
+  if (days < 3) {
+    number = days * 40;
+  } else if (days >= 3 && days < 7) {
+    number = days * 40 - 20;
+  } else if (days >= 7) {
+    number = days * 40 - 50;
```

Comment:
The use of literals in the expressions is not recommended as per the checklist. Define constants for the base price per day, the short term threshold, the long term threshold, and the respective discounts for better readability and maintainability.